MAVProxy/modules/mavproxy_console.py

```diff
@@ -514,6 +526,11 @@ def mavlink_packet(self, msg):
                 alt_error = "%d%s" % (msg.alt_error, alt_error_sign)
             self.console.set_status('AltError', 'AltError %s' % alt_error)
             self.console.set_status('AspdError', 'AspdError %.1f%s' % (msg.aspd_error*0.01, aspd_error_sign))
+        for i in range(0,len(self.user_added)):
```

Comment:
we only want to trigger on msgs that we know are in expressions the user has added, otherwise we waste a lot of cpu